src/types/expressSession.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserInfo {
+  email: string
+  familyName: string
+  givenName: string
+  roles?: Array<string | null> | null
```

Comment:
is there a case where a role will not be assigned to the user and hence will be null?

Reply:
At this very moment, that's the case. No roles are assigned by the system yet. When you login, then you have no role. And admin has to assign you a role.

Reply:
K, did address this. I made it non nullable